code/game/objects/items/weapons/weaponry.dm

```diff
@@ -54,6 +54,118 @@
 	user.visible_message("<span class='suicide'>[user] is falling on the [src.name]! It looks like \he's trying to commit suicide.</span>")
 	return(BRUTELOSS)
 
+var/list/highlander_claymores = list()
+/obj/item/weapon/claymore/highlander //ALL COMMENTS MADE REGARDING THIS SWORD MUST BE MADE IN ALL CAPS
+	desc = "<b><i>THERE CAN BE ONLY ONE, AND IT WILL BE YOU!!!</i></b>"
+	flags = CONDUCT | NODROP
+	block_chance = 0 //RNG WON'T HELP YOU NOW, PANSY
+	attack_verb = list("brutalized", "eviscerated", "disemboweled", "hacked", "carved", "cleaved", "gored") //ONLY THE MOST VISCERAL ATTACK VERBS
+	var/notches = 0 //HOW MANY PEOPLE HAVE BEEN SLAIN WITH THIS BLADE
+	var/announced = FALSE //IF WE ARE THE ONLY ONE LEFT STANDING
+
+/obj/item/weapon/claymore/highlander/New()
+	..()
+	START_PROCESSING(SSobj, src)
+	highlander_claymores += src
+
+/obj/item/weapon/claymore/highlander/Destroy()
+	STOP_PROCESSING(SSobj, src)
+	for(var/mob/living/simple_animal/shade/S in src)
+		S << "<span class='userdanger'>Your home breaks apart, and you with it!</span>"
+		qdel(S)
+	highlander_claymores -= src
+	return ..()
+
+/obj/item/weapon/claymore/highlander/process()
+	if(isliving(loc))
+		var/mob/living/L = loc
+		if(L.stat == DEAD)
+			L << "<span class='userdanger'>You are unworthy.</span>"
+			L.visible_message("<span class='warning'>[L] explodes in a shower of gore!</span>")
+			L.gib(TRUE)
+		else
+			if(announced)
+				return
+			for(var/obj/item/weapon/claymore/highlander/H in highlander_claymores)

```

Comment:
1. I don't see why not
2. Yes, but it would probably never happen so I didn't put in a failsafe. Highlander is an admin button and I doubt they'd press it without a lot of people.